src/components/Landingpage.js

```diff
@@ -60,7 +61,7 @@ function Landingpage() {
     <div>
       <NavHeader />
       <Slider />
-      <Content className="site-layout" style={{ padding: '0 50px', marginTop: 30, alignContent: 'center', textAlign: 'center', alignItems: 'center', justifyContent: 'center' }}>
+      <Content className="site-layout" style={{ padding: '0 100px', marginTop: 30, alignContent: 'center', textAlign: 'center', alignItems: 'center', justifyContent: 'center' }}>
```

Comment:
@ajay2k15 Why we have declared so much of inline CSS here, Then what is wrong with 'site-layout' class?